lib/pry/config/behavior.rb

```diff
@@ -175,6 +220,13 @@ def eager_load!
         end
       end
 
+      #
+      # @example
+      #   _pry_.config.last_default # Returns an instance of Pry::Config::Default
```

Comment:
This example doesn't show anything interesting, I'd rather remove it.

Reply:
i updated the example rather than remove it, can you take a look?